### PR TITLE
riff 0.4.0

### PR DESCRIPTION
**Pull request by fbiville**
Repository has moved and system commands have been removed.

-----

- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

Note: could not locally run `audit` nor `style` command due to https://github.com/Homebrew/brew/issues/5561.

### CODE REVIEW

File: Formula/riff.rb [hunk 1]
```diff
@@ -24,11 +24,18 @@ def install
     end
   end
 
+  def caveats
```
Comment (lembacon) [inline Formula/riff.rb]: Please remove these caveats, as these are not Homebrew-specific. See https://github.com/Homebrew/homebrew-core/pull/43053#issuecomment-521178021.

Comment (fbiville) [reply Formula/riff.rb]: Fixed, thanks!

**Comment by fbiville** [comment]
Now rebased.

**Comment by SMillerDev** [comment]
Thanks @fbiville! Without contributions like yours it'd be impossible to keep homebrew going with the high standards that users have come to expect from the project. You can feel good knowing that you've made the world a tiny bit better for homebrew users around the world! 👍 🎉

**Comment by SMillerDev** [comment]
@brewtestbot test this please!

**Pull request closed without merging**